code/modules/mob/living/say.dm

```diff
@@ -335,6 +335,11 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(!IsVocal())
 		return FALSE
 
+	if(isipc(src))
+		var/mob/living/carbon/target = src
+		if(!target.getorganslot(ORGAN_SLOT_TONGUE))
+			return FALSE  // who stole my soundblaster?!
```

Comment:
Should probably be part of IsVocal, preferably something contained inside the IPC species datum if that exists.

Reply:
alright..how do i put it in that datum

Reply:
Move it /mob/living/carbon/human/species/ipc as an override of IsVocal instead of in the general human one

Reply:
i dont- the only place /mob/living/.../ipc seems to exist is to set a datum and i tried adding it to the datum it crashed

Reply:
basic inheritance, you should OVERRIDE the speech proc on the ipc type instead of adding an istype blurb to the parent type of human. just find the ipc object and override isvocal to check the trait

Reply:
overriding in DM is just making a proc with the same name and without the /proc/ prefix
make sure you call ..()

Reply:
HOW DO I OVERRIDE IT ADDING IsVocal TO THE DATUM DOES NOTHING

Reply:
so like?? like /datum/species/ipc/IsVocal will change the human IsVocal proc?

Reply:
okay worked out an alternative on discord